src/ImageSharp/Processing/Processors/Quantization/EuclideanPixelMap{TPixel}.cs

```diff
@@ -200,24 +215,16 @@ public ColorDistanceCache(MemoryAllocator allocator)
         }
 
         [MethodImpl(InliningOptions.ShortMethod)]
-        public void Add(Rgba32 rgba, byte index)
+        public readonly void Add(Rgba32 rgba, byte index)
```

Comment:
I don't think the `readonly` is correct here, as it mutates the field `tablePointer`.

Below in `TryGetValue` it's fine, no mutation happens.